terminus-ui/selection-list/testing/src/test-components.ts

```diff
@@ -309,6 +309,36 @@ export class Seeded {
   }
 }
 
+@Component({
+  template: `
+    <ts-selection-list
+      [formControl]="myCtrl"
+      [reopenAfterSelection]="keepOpen"
+      [displayFormatter]="formatter"
+    >
+      <ts-option
+        *ngFor="let option of states"
+        [value]="option"
+        [option]="option"
+        [isDisabled]="option?.disabled"
+      >
+        <span tsOptionDisplay>
+          {{ option.name }}
+        </span>
+      </ts-option>
+    </ts-selection-list>
+  `,
+})
+export class SeededSingleSelect {
+  public states: State[] = STATES.slice();
+  public myCtrl = new FormControl([STATES[6]]);
+  public keepOpen = false;
+  public formatter: TsSelectionListFormatter | undefined;
+  public setFormatter() {
+    this.formatter = v => (v as State).population;
```

Comment:
Do we use this dynamic formatter setter anywhere? Or can we just statically define the formatter func?

Also, the `as State` coercion shouldn't be necessary if we type the param:

```typescript
formatter(state: State) {
  return state.population;
}
```